Updated MDK tutorial (#5376)
SAPDocuments/Tutorials: tutorials/cp-mobile-dev-kit-rest-api/cp-mobile-dev-kit-rest-api.md

    | Field | Value |
    |----|----|
    | `Destination Name` | `swagger.petstore` |
    | `URL` | `https://petstore.swagger.io/v2` |

    !![MDK](img_009.5.png)

8. For this tutorial, there is no Custom Headers, Annotations, Authentication required, click **Next** and Finish the form.

    In MDK rule, you will reference `Petstore` root URL via newly added destination `swagger.petstore`.

[DONE]
[ACCORDION-END]

[ACCORDION-BEGIN [Step 3: ](Create your Dev Space in SAP Business Application Studio)]

Ensure that you have already [Set Up SAP Business Application Studio for Development](appstudio-onboarding).

1. Choose **Create Dev Space**.

    !![MDK](img_0.0.png)

2. Choose `MDKTutorial` as the name for your dev space and **SAP Cloud Platform Mobile Services** as the application type. Continue with **Create Dev Space**.

    !![MDK](img_0.1.png)

> 3. When it is ready, open your dev space by clicking on the name.

Reviewer: We don't say "click on".

```suggestion
3. When it is ready, open your dev space by clicking its name.
```

    Double click the `NavToPet_Create.action` action and click **OK** to set it as the `OnPress` Action.

4. Save the changes to the `Main.page`.

[DONE]
[ACCORDION-END]

[ACCORDION-BEGIN [Step 17: ](Redeploy the application)]

Right-click the `Application.app` file in the project explorer pane and select **MDK: Deploy**.

[DONE]
[ACCORDION-END]

[ACCORDION-BEGIN [Step 18: ](Update the MDK app with new metadata)]

[OPTION BEGIN [Android]]

1. Re-launch the app on your device, authenticate with passcode or Touch ID if asked. You will see a _Confirmation_ pop-up, tap **OK**.

2. Tap **+** icon on `Main.page` to navigate to Create Pet page.

    !![MDK](img_9.png)

3.  Fill-up the details to create a new Pet record.

    !![MDK](img_10.1.1.png)

    A new record has been created and results are displaying in alert pop-up.

    !![MDK](img_11.png)

[OPTION END]

[OPTION BEGIN [iOS]]

1. Re-launch the app on your device, authenticate with passcode or Touch ID if asked. You will see a _Confirmation_ pop-up, tap **OK**.

2. Tap **+** icon on `Main.page` to navigate to Create Pet page.

    !![MDK](img_9.1.1.png)

> 3.  Fill-up the details to create a new Pet record.

Reviewer: BE:
```suggestion
3.  Fill in the details to create a new Pet record.
```
AE:
```suggestion
3.  Fill out the details to create a new Pet record.
```
